lib/fluent/plugin/out_influxdb.rb

```diff
@@ -90,25 +98,15 @@ def start
     end
   end
 
-  FORMATTED_RESULT_FOR_INVALID_RECORD = ''.freeze
-
-  def format(tag, time, record)
```

Comment:
Maybe, custom format is needed because influxdb plugins mutate record with `precision_time` and skip invalid record.